### PR TITLE
feat(ui): add device buttons control panel

### PR DESCRIPTION
**Pull request by oshcherbakovv**

### CODE REVIEW

**Comment by oshcherbakovv** [comment]
image

**Comment by oshcherbakovv** [comment]
image

File: ui/src/components/ui/device-control-panel/tabs/device-buttons-control/button-control.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  iconWidth?: number
+  iconHeight?: number
+  tooltipText: string
+  children?: string
+  className?: string
+  appearance?: 'accent' | 'neutral' | 'positive' | 'negative' | 'overlay' | 'accent-invariable'
+  onClick?: () => void
+}
+
+export const ButtonControl = ({
```
Comment (irdkwmnsb) [inline ui/src/components/ui/device-control-panel/tabs/device-buttons-control/button-control.tsx:20]: I think at one point we should document every component we have. Or make a stylebook. Not that important, but will be nice to have when more people will want to join the project

File: ui/src/store/device-control-store.ts [hunk 2]
```diff
@@ -38,6 +38,102 @@ export class DeviceControlStore {
     this.deviceControlService.back()
   }
 
+  pressPower(): void {
```
Comment (irdkwmnsb) [inline ui/src/store/device-control-store.ts]: I'm not sure we need to have so many methods with a single line that calls another function. Maybe we can use `new Proxy` to override attribute lookup and if it's a known attribute of deviceControlService - call it by `[]`. 

I'm worried adding new methods or buttons might be cumbersome

**Review by irdkwmnsb — APPROVED** [review]

**Pull request merged**